app/src/wpm.c

```diff
@@ -72,7 +72,7 @@ void wpm_expiry_function() { k_work_submit(&wpm_work); }
 
 K_TIMER_DEFINE(wpm_timer, wpm_expiry_function, NULL);
 
-int wpm_init() {
+static int wpm_init(void) {
```

Comment:
Why do this become static?

Reply:
For consistency, it should have been static in the first place, it doesn't need to be an exported public symbol.